### DISCUSSION TITLE
Change gamma only for white color

### DISCUSSION THREAD

**Discussion by luca-demma** [zb3/gnome-gamma-tool#8] [Q&A]
Hi guys :smiley: 

I have a problem with my laptop screen, the white color is too bright compared to the other colors of the screen, so when I change from a dark window (i.e. vscode) to a bright window (whatever website with no dark theme), I have to change the brightness because it's uncomfortable. 

Is there a way to change the gamma value only for the white color , instead of the r-g-b values?

Thanks

**Comment by zb3** [comment]
Well, the white color means all RGB components are set to their maximum value, hence I can't really understand what you mean.

It seems to me that you should increase your gamma (like with the `-g 1.1` switch). This will affect all colors and I recommend you experiment with different values.

You should also try changing contrast, that's another setting which affects all colors.

Other color transformations (based on hue) are not possible due to VCGT limitations.
